src/main/java/org/apache/bcel/generic/LDC.java

```diff
@@ -89,6 +89,8 @@ public Type getType(final ConstantPoolGen cpg) {
             return Type.INT;
         case org.apache.bcel.Const.CONSTANT_Class:
             return Type.CLASS;
+        case org.apache.bcel.Const.CONSTANT_Dynamic:
```

Comment:
Warning->Tangent: I wonder why we case Float but not Double.

Reply:
For a Double (or a Long) the instruction would be `LDC2_W` I think, because it is wider